Core.Collectors/Web/RequestResult.cs

```diff
@@ -6,19 +6,20 @@ namespace Microsoft.CloudMine.Core.Collectors.Web
 {
     public class RequestResult
     {
+        public HttpRequestMessage request;
         public HttpResponseMessage response;
         public IAllowListStatus allowListStatus;
 
-        public RequestResult(HttpResponseMessage response)
-                : this(response, null)
+        public RequestResult(HttpRequestMessage requestMessage, HttpResponseMessage response)
```

Comment:
>HttpResponseMessage response [](http://example.com/codeflow?start=64&length=28)

Just wondering: doesn't response already have the request? If you can get the request (URL) from response, then you don't need to pass the request again, no?

Reply:
The response does contain the request, however in the case of an allow-listed exception there will not be a response for us to use.

Reply:
I have changed this constructor to take only an HttpResponseMessage and then filled both request and response fields using that object.